Cargo.toml

```diff
@@ -18,6 +18,14 @@ members = [
 default = ["networking"]
 networking = ["reqwest"]
 
+[package.metadata.bundle]
+name = "Emulsion"
+identifier = "io.github.arturkovacs.emulsion"
+icon = ["resource_dev/emulsion.png"]
+version = "1.9"
+copyright = "Creative Commons Attribution 3.0 License"
```

Comment:
Version: Does the MacOS build fail if this is not specified? If it's possible to not specify the verion here but used the one at the `[package]` section it would be better. I'm wondering because with this, the Cargo.toml file declares the version twice which must always be updated togeter, which is error prone.

Copyright: Emulsion has an MIT license currently. See the LICENSE.txt